sklearn/tree/tests/test_tree.py

```diff
@@ -2417,6 +2417,44 @@ def test_missing_values_on_equal_nodes_no_missing(criterion):
     assert_allclose(y_pred, [np.mean(y_equal[-4:])])
 
 
+@pytest.mark.parametrize("seed", range(3))
+@pytest.mark.parametrize("criterion", ["squared_error", "friedman_mse"])
+def test_missing_values_random_splitter_on_equal_nodes_no_missing(criterion, seed):
+    """Check missing values go to the correct node during predictions for ExtraTree.
+
+    Since ETC use random splits, we use different seeds to verify that the
+    left/right node is chosen correctly when the splits occur.
+    """
+    X = np.array([[0, 1, 2, 3, 8, 9, 11, 12, 15]]).T
+    y = np.array([0.1, 0.2, 0.3, 0.2, 1.4, 1.4, 1.5, 1.6, 2.6])
+
+    etr = ExtraTreeRegressor(random_state=seed, max_depth=1, criterion=criterion)
+    etr.fit(X, y)
+
+    # see which node has the most data points
+    etr.tree_.value
```

Comment:
Is this value used because it doesn't seem to be stored?

Reply:
No, I removed it in [fea6cbf](https://github.com/scikit-learn/scikit-learn/pull/27966/commits/fea6cbf42534a7553644e4d06117d561924e6eeb)